en/docs/deploy/deploy-api/grouping-apis-with-vhosts.md

```diff
@@ -0,0 +1,98 @@
+# Grouping APIs with Virtual Hosts
```

Comment:
As the title of the page has changed we need to change the name of the page as well.